dependency-management/dependency-management-impl/src/main/java/io/cloudslang/dependency/impl/services/DependencyServiceImpl.java

```diff
@@ -200,6 +233,37 @@ private void buildDependencyFile(String[] gav) {
             throw new IllegalStateException(fileToReturn.getPath() + " not found");
         }
         appendSelfToPathFile(gav, fileToReturn);
+        sendMavenDependencyBuildFinishedEvent(gav);
+    }
+
+    private void sendMavenDependencyBuildFinishedEvent(String[] gav) {
+        String message = "Artifact download complete";
+        if (gav.length >= 3) {
+            message = String.format("Downloading complete for artifact with gav: %s:%s:%s ", gav[0], gav[1], gav[2]);
+        }

```

Comment:
I think it is better to have a single line for each artifact (either `Downloading `or `Download successful`) since failures will be logged anyway it might make it difficult to trace real problems in the log. WDYT?